src/js/angular/export/controllers.js

```diff
@@ -286,13 +277,7 @@ exportCtrl.controller('ExportCtrl',
                 }).result
                     .then(function () {
                         $timeout(function () {
-                            const url = 'repositories/' + $repositories.getActiveRepository() + '/statements';
-                            $http({
-                                method: 'POST',
-                                url: url,
-                                data: 'update=CLEAR ALL',
-                                headers: {'Content-Type': 'application/x-www-form-urlencoded'}
-                            })
+                            RDF4JRepositoriesRestService.addStatements($repositories.getActiveRepository(), 'update=CLEAR ALL')
```

Comment:
That's not adding 😄 maybe `RDF4JRepositoriesRestService.updateStatements` ?

Reply:
Yeah, but as long as the actual operation comes as parameter it might not update I guess